Makefile

```diff
@@ -5,14 +5,24 @@ FULL_GOPATH=${GOPATH}/src/${FULL_NAME}
 
 default: test
 
-get:
-	go get "github.com/omeid/go-resources/cmd/resources"
+dependencies:	
 	go get -t -v ./...
 
-build: get
-	cd example/agent-ping && resources -output="resources.go" -var="Resources" -trim="../" resources/* ../schema/*
-	cd example/agent-pong && resources -output="resources.go" -var="Resources" -trim="../" resources/* ../schema/*
+build: dependencies	
 	go build -v ./...
 
+examples: dependencies
```

Comment:
Ok. I see. _make_ install the binaries of the examples. So, agree with you, the generation of the resources must be performed.

I would prefer to have 2 separate phases, one for building the lib, another one for building the examples, if wanted. But I don't sure it is feasible (easily in any case).